cmd/serverless/main.go

```diff
@@ -296,6 +288,16 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 
 	wg.Wait()
 
+	// starts otlp agent
+	// TODO: OTLP agent setup should only wait for logs-agent if logs are enabled. Otherwise, should start in goroutine
+	if !otlp.IsEnabled() {
+		log.Debug("otlp endpoint disabled")
+	} else {
+		otlpAgent := otlp.NewServerlessOTLPAgent(metricAgent.Demux.Serializer(), logsAgentChannel)
```

Comment:
So, I don't think we want to enable logs collection in OTLP when running in serverless. AWS lambda has a whole [different mechanism](https://docs.aws.amazon.com/lambda/latest/dg/telemetry-api.html) for us to receive logs from customer lambdas. Until we hear otherwise from our customers, I would like for us to have logs collection disabled in otlp on serverless.